README.md

```diff
@@ -186,9 +186,25 @@ const credential = await navigator.credentials.get({
 });
 ```
 
-### Authenticating a payment
+## Confirming a payment
 
-Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) with the `secure-payment-confirmation` payment method to prompt the user to verify a `PaymentCredential` created by any other origin. The `PaymentRequest.show()` method requires a user gesture. The browser will display a native user interface with the payment amount and the payee origin, which is taken to be the origin of the top-level context where the `PaymentRequest` API was invoked.
+Any origin may invoke the [Payment Request API] in order to use a PaymentCredential to confirm a payment. The caller will either:
+ - invoke the API using a known `instrumentId` (using the `secure-payment-confirmation` payment method identifier), or
+ - invoke the API using one or more payment methods that support a Secure Payment Confirmation response and have a supporting Payment Handler installed
+
+**Note:** The `PaymentRequest.show()` method requires a user gesture.
+
+### Using the `secure-payment-confirmation` payment method
+
+Any origin can use the [Payment Request API] with the `secure-payment-confirmation` payment method to immediately prompt the user to use a `PaymentCredential` to confirm a payment.
+
+In this case the caller must provide:
+ - an `instrumentId` provided by the RP for a payment instrument that was already selected by the user
+ - an opaque `challenge` provided by the RP
```

Comment:
Hi @adrianhopebailie, this shouldn't be necessary: we're hashing the `paymentData` to get the challenge and there's nothing in there that has to be provided by the RP.

Reply:
We can't invoke WebAuthn without some dynamic data from the RP.